Kerdy/Kerdy/Filter/Views/DateSelectBtnView.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  DateSelectBtnView.swift
+//  Kerdy
+//
+//  Created by 이동현 on 11/5/23.
+//
+
+import UIKit
+
+final class DateSelectBtnView: UIView {
+    lazy var imageView: UIImageView = {
+        let imageView = UIImageView()
+        return imageView
+    }()
+
+    lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = nil
+        label.font = .nanumSquare(to: .regular, size: 13)
+        return label
+    }()
+
+    lazy var button: UIButton = {
+        let button = UIButton()
+        button.backgroundColor = .clear
+        button.setTitle(nil, for: .normal)
+        return button
+    }()
+
+    init() {
+        super.init(frame: .zero)
+        setLayout()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setLayout() {
+        addSubviews(imageView)
+        addSubviews(titleLabel)
+        addSubviews(button)
+
+        imageView.snp.makeConstraints {
+            $0.width.equalTo(14)
+            $0.height.equalTo(14)
```

Comment:
width와 height이 동일하다면 size로 표현 가능합니다 : )
```suggestion

            $0.size.equalTo(14)
```